### PR TITLE
General: Add a security policy

### PR DESCRIPTION
**Pull request by kraftbj**
[GitHub will highlight](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository) a SECURITY.md file when new issues are created to help notify folks of where to report security issues.

This initial form was borrowed and modified from WordPress' SECURITY.md file. Since our policy is on HackerOne, generally, the policy will direct folks to there to avoid having duplicative information in multiple locations.

#### Changes proposed in this Pull Request:
* Adds a security policy.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
None.

#### Proposed changelog entry for your changes:
* None needed.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Our most critical targets are:
+
+* Jetpack and the Jetpack composer packages (all within this repo)
+* Jetpack.com -- the primary marketing site.
+* cloud.jetpack.com -- a management site.
+* wordpress.com -- the shared management site for both Jetpack and WordPress.com sites.
+
+For more targets, see the `In Scope` section on [HackerOne](https://hackerone.com/automattic).
+
+_Please note that the **WordPress software is a separate entity** from Automattic. Please report vulnerabilities for WordPress.com or the WordPress mobile apps through [the WordPress Foundation's HackerOne page](https://hackerone.com/wordpress)._
```
Comment (jeherve) [inline SECURITY.md:22]: Should this be without the .com?

```suggestion
_Please note that the **WordPress software is a separate entity** from Automattic. Please report vulnerabilities for WordPress or the WordPress mobile apps through [the WordPress Foundation's HackerOne page](https://hackerone.com/wordpress)._
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Security Policy
+
+Full details of the Automattic Security Policy can be found on [HackerOne](https://hackerone.com/automattic).
+
+## Supported Versions
+
+Generally, only the latest version of Jetpack has continued support. If a critical vulnerability is found in the current version of Jetpack, we may opt to backport any patches to previous versions. 
+
+## Reporting a Vulnerability
```
Comment (jeherve) [inline SECURITY.md:9]: Should we add a paragraph asking folks to report vulnerabilities at HackerOne, not in this repo? I find this missing from this document.

**Review by jeherve — CHANGES_REQUESTED** [review]
Can we move this to the `.github` directory with the other GitHub files? It seems to be supported, and would allow us to ensure this doesn't get shipped in the stable version of the plugin.

**Comment by kraftbj** [comment]
I thought about that, but seeing our code of conduct file in the root also made me think the increased visibility of it is a good thing. 

Even shipping in the stable version, white hat hackers (is there a better term for that now?) would see it.

What do you think?

**Comment by jeherve** [comment]
That's true. Let's keep it at the root then 👍

**Comment by kraftbj** [comment]
I'll post for secops to have a chance to weigh in.

File: SECURITY.md (hunk 2 above)
Comment (kraftbj) [reply SECURITY.md:9]: :y: f8a989a76

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+[Jetpack](https://jetpack.com/) is an open-source plugin for WordPress. Our HackerOne program covers the plugin software, as well as a variety of related projects and infrastructure.
+
+**For responsible disclosure of security issues and to be eligible for our bug bounty program, please submit your report via the  [HackerOne](https://hackerone.com/automattic) portal.**
```
Comment (jeherve) [inline SECURITY.md:13]:
```suggestion
**For responsible disclosure of security issues and to be eligible for our bug bounty program, please submit your report via the [HackerOne](https://hackerone.com/automattic) portal.**
```

**Review by jeherve — CHANGES_REQUESTED** [review]

**Review by jeherve — DISMISSED** [review]

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Our most critical targets are:
+
+* Jetpack and the Jetpack composer packages (all within this repo)
+* Jetpack.com -- the primary marketing site.
+* cloud.jetpack.com -- a management site.
+* wordpress.com -- the shared management site for both Jetpack and WordPress.com sites.
+
+For more targets, see the `In Scope` section on [HackerOne](https://hackerone.com/automattic).
+
+_Please note that the **WordPress software is a separate entity** from Automattic. Please report vulnerabilities for WordPress or the WordPress mobile apps through [the WordPress Foundation's HackerOne page](https://hackerone.com/wordpress)._
```
Comment (xknown) [inline SECURITY.md:24]:
```suggestion
_Please note that the **WordPress software is a separate entity** from Automattic. Please report vulnerabilities for WordPress through [the WordPress Foundation's HackerOne page](https://hackerone.com/wordpress)._
```

**Review by jeherve — APPROVED** [review]

**Pull request merged**